### PR TITLE
Browser: Add GraphQL option for grouping segments

### PR DESCRIPTION
**Pull request by rhetoric101**

### CODE REVIEW

File: src/content/docs/browser/new-relic-browser/configuration/group-browser-metrics-urls.mdx [hunk 1]
```diff
@@ -39,11 +39,10 @@ You can modify how browser groups your data by adding to your application's URL
 
 Order is not important when listing multiple segments. However, make sure the segments you add **exactly match** the way the segments appear in your URLs. If a URL path segment ends in `.html` or `.jsp`, the extension must be included in the segment. Segments are case-sensitive.
 
-To add new URL segments to your application's allow list:
+To add new URL segments to your application's allow list, you can add them programmatically using [GraphQL](/docs/apis/nerdgraph/examples/browser-monitoring-config-nerdgraph#browser-segments), or you can use segment allow lists in the New Relic browser monitoring UI like this:
```
Comment (jeff-colucci) [inline src/content/docs/browser/new-relic-browser/configuration/group-browser-metrics-urls.mdx:42]: I'm not sure the "like this" is completely necessary here, but it's just a small and probably inconsequential change to make

**Review by jeff-colucci — DISMISSED** [review]
Looks great to me!

File: src/content/docs/apis/nerdgraph/examples/browser-monitoring-config-nerdgraph.mdx [hunk 2]
```diff
@@ -214,8 +216,90 @@ query FetchBrowserConfiguration($guid: EntityGuid!) {
...
+          ]
+        }
+      }
+    }
+  }
+```
+
+### Create segments [#create-segments]
+
+Create browser segments using the `agentApplicationSegmentsReplaceAllBrowserSegmentAllowList` mutation:
```
[the remark this answers is not in the document]
Comment (rhetoric101) [reply src/content/docs/apis/nerdgraph/examples/browser-monitoring-config-nerdgraph.mdx:278]: @jeff-colucci, "mutation" is the standard word in GraphQL that describes the option to make changes  (create, update, or delete).

File: src/content/docs/browser/new-relic-browser/configuration/group-browser-metrics-urls.mdx (hunk 1 above)
Comment (rhetoric101) [reply src/content/docs/browser/new-relic-browser/configuration/group-browser-metrics-urls.mdx:42]: @jeff-colucci good catch! I dropped that!

**Pull request merged**